.github/workflows/release.yml

```diff
@@ -13,26 +13,20 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-        
-      - name: Read version.properties
-        id: version
-        uses: juliangruber/read-file-action@v1
-        with:
-          path: ./version.properties
-        
-      - uses: rishabhgupta/split-by@v1
-        id: split
+
+      - uses: madhead/read-java-properties@latest
```

Comment:
can we keep `name: Read version.properties` as well?

Reply:
@doannimble so you skipped this review?

Reply:
@luongvo Opps, I didn't see it.
Of course yes, let me put the name as a description.

Reply:
@luongvo I've updated.